### DISCUSSION TITLE
Celery worker aborts without running

### DISCUSSION THREAD

**Discussion by Harut8** [celery/celery#9189] [Q&A]
I have some issue with celery and k8s in AWS EKS.  In my local env (docker, minikube) when i tried to deploy celery app as worker with dockerfile bellow
`FROM python:3.11
WORKDIR /app
COPY pyproject.toml poetry.lock ./
RUN pip install --upgrade pip
RUN pip --no-cache-dir install poetry
RUN poetry config virtualenvs.create false
RUN poetry install
ADD .. /app/
RUN groupadd -r celery && useradd -r -g celery celery
USER celery
CMD ["celery", "-A", "app.task_manager.celery_app", "worker","--pool=solo","--without-gossip", "--events", "-Ofair", "--loglevel=debug", "--concurrency=1"]`

All is working fine, celery worker was running. But when i have deployed it in EKS with helm charts  it's not running with any logs. 
i have tired to see logs, describe pod get events. but there are not some errors or something else  also behind this, the pod where it's running gets status.  CrashLoopBackOff and restart pod immediately 
I checked Rabbitmq(broker) url. It's okay

I tried with entrypoint, cmd. with any other celery configuration  but it still not working and there are not errors.

Also when i ran it with sudo command. like sudo celery -A ...  it runs but then it can not read secrets from k8s

**Comment by thedrow** [comment]
This is likely not a celery issue. I'm converting this to a discussion, maybe someone will have some insights on why you're having problems with your setup.
